benchmark/child_process/spawn-echo.js

```diff
@@ -1,12 +1,12 @@
 'use strict';
 var common = require('../common.js');
 var bench = common.createBenchmark(main, {
-  thousands: [1]
+  len: [1000]
```

Comment:
Yes, we should be more consistent and use `n` where possible. There are some benchmarks like this and others that use `thousands` or `millions` though. I prefer to just use `n`, even if the values are large, for consistency.